.vscode/extensions.json

```
@@ -2,7 +2,7 @@
   // See http://go.microsoft.com/fwlink/?LinkId=827846
   // for the documentation about the extensions.json format
   "recommendations": [
-    "esbenp.prettier-vscode",
+    "biomejs.dev.biomejs.biome",
```

```suggestion
    "biomejs.biome",
```
that one extension id [doesn't seem to resolve to anything](https://marketplace.visualstudio.com/items?itemName=biomejs.dev.biomejs.biome)

---

Nice catch. Thank you!